### DISCUSSION TITLE
Mage Training Arena - Telekinetic Room - Still Not Working.

### DISCUSSION THREAD

**Discussion by FayeDaBeast** [runelite/runelite#15754] [Q&A] [answered]
Hello, I am having the same issue regarding the telekinetic room. I have tried to fix this client several attempts and unable for the indicator to get past the first marking on a puzzle. Thank you!
[Document.docx](https://github.com/runelite/runelite/files/9681138/Document.docx)

**Comment by Nightfirecat** [comment, marked as answer]
Duplicate of #15741. This has been fixed in 6af244848326e2283c3a9a966b50d2bafa3c837e, which we will likely release in an update tomorrow.

**Reply by FayeDaBeast** [reply]
Thank you for ETA.
